JS/Planes/Plane.js

```diff
@@ -1,28 +1,22 @@
 class Plane {
```

Comment:
You still have Planes folder written with capital letter. Please correct.

Reply:
https://github.com/VladSelikh/aircompany/commit/f42aba0d968860bb4fe32c57bacd1778fa929c81

Reply:
https://github.com/VladSelikh/aircompany/commit/f42aba0d968860bb4fe32c57bacd1778fa929c81